parachain/src/chain_spec.rs

```diff
@@ -354,6 +354,11 @@ fn testnet_genesis(
         technical_membership: Default::default(),
         democracy: Default::default(),
         elections_phragmen: Default::default(),
+        supply: SupplyConfig {
+            initial_supply: token_distribution::INITIAL_ALLOCATION,
+            start_height: YEARS * 5,
+            inflation: FixedU128::checked_from_rational(102, 100).unwrap(), // 1.02%
```

Comment:
I didnt look at the code, but are you sure the comment is right? Shouldn't it be 2%?

Reply:
Yeah I was unnecessarily subtracting 100% to get the total inflation.